### PR TITLE
Update app-templates.md to reflect retirement of certain app templates

### PR DESCRIPTION
**Pull request by shuchauh**
Updating app-templates.md to reflect retirement of apps based on Poll, Survey, Checklist, Quiz and Training templates. Link to active MC post request - https://m365-mcrp.azurewebsites.net/request/active/10549.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/samples/app-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/14dd7ea539092a121b891694083b2c4d359b84ba/msteams-platform%2Fsamples%2Fapp-templates.md) | 92 | 47 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3b1d4c93-efb2-4f0c-b82b-8b0e84afe856) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/samples/app-templates.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ App template code samples are a collection of sample apps for Microsoft Teams pl
 >
 > * Microsoft is not actively managing the source code. The complete source code is open source and you can explore, fork, and modify the source code to meet your specific requirements.
 > * You must not use Microsoft Power Platform to create apps that are to be published to the Teams app store. Microsoft Power Platform apps can be published to an organization’s app store only.
+> * Applications based on Poll, Survey, Checklist, Quiz and Training templates will be retired soon. The applications will stay active until February and users will be able to retrieve any data until early March. By the end of March the applications will be fully retired.
```
Comment (surbhigupta12) [inline msteams-platform/samples/app-templates.md:99]:
```suggestion
> * Applications based on Poll, Survey, Checklist, Quiz, and Training templates will be retired completely by March 31, 2023. The applications will stay active until February 28, 2023 and you will be able to retrieve any data until early March. 
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/samples/app-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/f1a53c7cd79b6a134ecc1e924d600ba7db065a48/msteams-platform%2Fsamples%2Fapp-templates.md) | 90 | 56 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/da647350-99f0-426d-9368-f3f95ec5c08f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/samples/app-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/669e8bafd096b1f52e6878b692743ebaf4e06abe/msteams-platform%2Fsamples%2Fapp-templates.md) | 90 | 56 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d93d2434-9b0c-4b2f-a251-7090201943ce) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/samples/app-templates.md [hunk 2]
```diff
@@ -109,7 +110,7 @@ The following table describes App template code samples:
...
-| Checklist | Checklist is a custom Teams [message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to collaborate with your team by creating a shared checklist in a chat or channel. The app is supported across all Teams platform clients, such as desktop browser, iOS, and Android. The app is ready for deployment as part of your Microsoft 365 subscription.  | [Checklist](https://github.com/OfficeDev/microsoft-teams-checklist-app)|
+| Checklist | Note: The Checklist App Template will be retired soon. For more details refer the note section [here]([url](https://learn.microsoft.com/en-us/microsoftteams/platform/samples/app-templates#app-template-code-samples)). <br/> Checklist is a custom Teams [message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to collaborate with your team by creating a shared checklist in a chat or channel. The app is supported across all Teams platform clients, such as desktop browser, iOS, and Android. The app is ready for deployment as part of your Microsoft 365 subscription.  | [Checklist](https://github.com/OfficeDev/microsoft-teams-checklist-app)|
```
Comment (shuchauh) [inline msteams-platform/samples/app-templates.md:113]:
```suggestion
| Checklist | Note: The Checklist App Template will be retired soon. For more details refer the note section [here]([url](../samples/app-templates#app-template-code-samples)). <br/> Checklist is a custom Teams [message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to collaborate with your team by creating a shared checklist in a chat or channel. The app is supported across all Teams platform clients, such as desktop browser, iOS, and Android. The app is ready for deployment as part of your Microsoft 365 subscription.  | [Checklist](https://github.com/OfficeDev/microsoft-teams-checklist-app)|
```

File: msteams-platform/samples/app-templates.md [hunk 3]
```diff
@@ -132,9 +133,9 @@ The following table describes App template code samples:
...
+|Quiz | Note: The Quiz App Template will be retired soon. For more details refer the note section [here]([url](https://learn.microsoft.com/en-us/microsoftteams/platform/samples/app-templates#app-template-code-samples)). <br/> Quiz is a custom [Teams message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to create a quiz within a chat or a channel for knowledge check and instantaneous results. You can use Quiz for, In-class and offline exams, Knowledge check within team, and for fun quizzes within a team. Quiz app is supported across multiple platforms, such as Teams desktop, browser, iOS, and Android clients. This app is ready for deployment as part of your existing Microsoft 365 subscription.|[Quiz](https://github.com/OfficeDev/microsoft-teams-apps-quiz) |
```
Comment (shuchauh) [inline msteams-platform/samples/app-templates.md:138]:
```suggestion
|Quiz | Note: The Quiz App Template will be retired soon. For more details refer the note section [here]([url](../samples/app-templates#app-template-code-samples)). <br/> Quiz is a custom [Teams message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables you to create a quiz within a chat or a channel for knowledge check and instantaneous results. You can use Quiz for, In-class and offline exams, Knowledge check within team, and for fun quizzes within a team. Quiz app is supported across multiple platforms, such as Teams desktop, browser, iOS, and Android clients. This app is ready for deployment as part of your existing Microsoft 365 subscription.|[Quiz](https://github.com/OfficeDev/microsoft-teams-apps-quiz) |
```

File: msteams-platform/samples/app-templates.md [hunk 4]
```diff
@@ -144,9 +145,9 @@ The following table describes App template code samples:
...
+| Training | Note: The Training App Template will be retired soon. For more details refer the note section [here]([url](https://learn.microsoft.com/en-us/microsoftteams/platform/samples/app-templates#app-template-code-samples)). <br/> Training is a custom [Teams message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables users to publish a training within a chat or a channel for offline knowledge sharing and upskilling. The app is supported across multiple Teams platform clients, such as desktop, browser, iOS, and Android. This app is ready for deployment as part of your Microsoft 365 subscription.|[Training](https://github.com/OfficeDev/microsoft-teams-apps-training) |
```
Comment (shuchauh) [inline msteams-platform/samples/app-templates.md:150]:
```suggestion
| Training | Note: The Training App Template will be retired soon. For more details refer the note section [here]([url](../samples/app-templates#app-template-code-samples)). <br/> Training is a custom [Teams message extension](../messaging-extensions/what-are-messaging-extensions.md) app that enables users to publish a training within a chat or a channel for offline knowledge sharing and upskilling. The app is supported across multiple Teams platform clients, such as desktop, browser, iOS, and Android. This app is ready for deployment as part of your Microsoft 365 subscription.|[Training](https://github.com/OfficeDev/microsoft-teams-apps-training) |
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/samples/app-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d29cf2e63354f9f386036a806b0f409668b3537e/msteams-platform%2Fsamples%2Fapp-templates.md) | 90 | 56 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/33436bec-5480-4593-8220-99cf05280b1d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/samples/app-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/19309289a713b4f838033b295a5b0768157a77fc/msteams-platform%2Fsamples%2Fapp-templates.md) | 90 | 56 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62344fda-05b7-455c-9e97-a951f525c4f6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/samples/app-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/c8111a50e8eaa4affba5e0a8b4fcc2892130cbe1/msteams-platform%2Fsamples%2Fapp-templates.md) | 90 | 56 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2572884e-80ee-42ac-8acb-28f190aa64b4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/samples/app-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a37dcb9e41a9831caff5010cf05991ddb1d8c2f7/msteams-platform%2Fsamples%2Fapp-templates.md) | 90 | 56 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/66c715e0-5fb6-4b21-8d5e-853e796f6534) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/samples/app-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e6c2432b56e4325d6a2c67b4a17a862841a2d9bf/msteams-platform%2Fsamples%2Fapp-templates.md) | 90 | 56 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8ea17aa6-9e2e-4772-b68f-3f6cd1f41fb4) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/samples/app-templates.md](https://github.com/MicrosoftDocs/msteams-docs/blob/ee74302f4416adc9dcc1f5d390024a297ed9f78b/msteams-platform%2Fsamples%2Fapp-templates.md) | 90 | 56 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ffe54321-ea28-4cce-ba71-63b2760a7a51) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**